apps/web/lib/video/[uid]/getServerSideProps.ts

```diff
@@ -161,6 +191,11 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
       calVideoLogo,
       displayLogInOverlay,
       loggedInUserName: sessionUserId ? session?.user?.name : undefined,
+      showRecordingButton,
+      redirectUrlOnExit:
+        sessionUserId !== bookingObj.user?.id
+          ? bookingObj.eventType?.calVideoSettings?.redirectUrlOnExit
```

Comment:
It is not clear what the purpose of `redirectUrlOnExit` is. Is it meant for hosts or guests or for everyone.

Reply:
We should rename `redirectUrlOnExit` to something like `rediectAttendeeToOnExit`

Reply:
Okay